src/components/styles/PageSection.styled.ts

```diff
@@ -13,34 +13,51 @@ const fadeInUp = keyframes`
     }
 `
 
-export const MainContainer = styled.div<{ hasImage: boolean }>`
-  margin: 75px 0;
+export const MainContainer = styled.div<{ hasImage: boolean; mayNeedReverse?: boolean }>`
+  margin-top: 20px;
+  margin-bottom: 75px;
   display: flex;
   padding: 0 50px 0 0;
   justify-content: space-between;
   background-color: black;
   animation-duration: 1s;
   animation-name: ${fadeInUp};
+
+  @media screen and (max-width: 800px) {
+    flex-direction: ${(props) => (props.mayNeedReverse ? 'column-reverse' : 'column')};
+    align-items: center;
+  }
 `
 
-export const ImageContainer = styled.div<{ image: string; textPosition: string }>`
-  height: 400px;
-  width: 400px;
+export const ImageContainer = styled.div<{ image: string; textPosition?: 'left' | 'right'; events?: boolean }>`
+  max-width: 350px;
+  min-height: 300px;
+  width: ${(props) => (props.events ? '20%' : '50%')};
   background-size: contain;
   background-repeat: no-repeat;
   background-position: center;
-  ${(props) => (props.textPosition === 'right' ? 'margin-left: 100px' : 'margin-right: 20px')};
   ${(props) => (props.image === '' ? `background-image: url(${Null});` : `background-image: url(${props.image});`)}
+
+  @media screen and (min-width: 800px) {
+    ${(props) => (props.textPosition === 'right' ? 'margin-left: 100px' : 'margin-right: 20px')};
+  }
+
+  @media screen and (max-width: 800px) {
+    margin: 50px 0;
+  }
 `
 
 export const TextContainer = styled.div<{ hasImage: boolean }>`
-  height: width: ${(props) => (props.hasImage ? '50%' : '100%')};
-  width: ${(props) => (props.hasImage ? '50%' : '83%')};
   margin-left: 8%;
   display: flex;
   flex-direction: column;
   justify-content: center;
   over-flow: auto;
```

Comment:
is there a over-flow property? should it be overflow? and should the text overflow?

Reply:
resolved in 48c1901